src/e2e/puppeteer/helpers/dragAndDropThought.ts

```diff
@@ -0,0 +1,35 @@
+import { Page } from 'puppeteer'
+import getEditable from './getEditable'
+
+/** PErforms Drag and Drop functionality on a hought in Puppeteer browser. */
+const dragAndDropThought = async (page: Page, sourceValue: string, destValue: string) => {
```

Comment:
You left out the parameter `{ position: 'after' | 'before' | 'child' }` that was suggested in https://github.com/cybersemics/em/pull/2038#discussion_r1641218166. Was this intentional?

You don't have to implement each of the positions yet, but we may as well set it up with the desired API now. The semantics are ambiguous without it.

Reply:
So can I just add the parameter for now, and we can get back to it at some later point?

Reply:
Yes, add the parameter and `throw` on the unsupported options for now.